build/git.go

```diff
@@ -111,5 +114,18 @@ func getGitAttributes(ctx context.Context, contextPath string, dockerfilePath st
 		}
 	}
 
-	return
+	return res, func(so *client.SolveOpt) {
+		if root == "" {
```

Comment:
When will this be true?

Reply:
If `BUILDX_GIT_LABELS=1` which is opt-in: https://github.com/docker/buildx/blob/8484fcdd5715c0683e8605e05cdffc0bfa8fde94/build/git.go#L24-L31

Reply:
you mean `BUILDX_GIT_INFO` ? I think if `BUILDX_GIT_INFO=0` then these paths should not be sent.

Reply:
No I mean `BUILDX_GIT_LABELS` because root dir is computed in https://github.com/docker/buildx/blob/8484fcdd5715c0683e8605e05cdffc0bfa8fde94/build/git.go#L97

Reply:
This isn't related to labels, so should be opt-out with the same var that controls URL/commit.